### DISCUSSION TITLE
Preview Options

### DISCUSSION THREAD

**Discussion by minh-trancong** [sbarex/QLMarkdown#96] [Q&A]
Hello,
Is it possible to view markdown files in the view column like the last image?

<img width="790" alt="image" src="https://user-images.githubusercontent.com/92774562/151350569-4319bc5d-6778-4fcc-aa68-9c3a866b27e5.png">

I hope we can preview in the column view like this one.

<img width="798" alt="image" src="https://user-images.githubusercontent.com/92774562/151350782-c78ecc9d-86a1-4dc1-9c7d-b3aee8f41781.png">

**Comment by sbarex** [comment]
In column view, the last column does not use the Quick Look preview but only the file icon. This app does not provide the thumbnail of the icon for the moment. And even if it did, it would not be possible to scroll through the content since it would be an icon.

**Reply by minh-trancong** [reply]
Oh yes, unfortunately. Thank you for your response.
